src/Tribe/Main.php

```diff
@@ -86,7 +86,6 @@ public function init_libraries() {
 
 		require_once $this->plugin_path . 'src/functions/template-tags/general.php';
 		require_once $this->plugin_path . 'src/functions/template-tags/date.php';
-		require_once $this->plugin_path . 'src/functions/template-tags/day.php';
```

Comment:
Correct. That file is also removed in this commit. The contents of the file was:

```php
<?php
```